### PR TITLE
Improve OG metadata

### PR DESCRIPTION
**Pull request by VeckoTheGecko**
Works towards #42 adding `description`, `og:description` and `og:image` metadata tags.

Built the website to ensure that `og:description` and `og:image` were set correctly. Feel free to push edits (perhaps choosing another preview image).

### CODE REVIEW

**Comment by VeckoTheGecko** [comment]
My changes to `_config.yml` were informed by [this article](https://mademistakes.com/mastering-jekyll/site-url-baseurl/#jekyll-development-and-siteurl:~:text=So%20what%20exactly%20are%20the%20url%20and%20baseurl%20variables%3F%20To%20start%2C%20both%20are%20site%2Dwide%20variables%20set%20in%20the%20_config.yml%20file%20and%20affect%20how%20Jekyll%20builds%20URLs).

File: _config.yml [hunk 1]
```diff
@@ -1 +1,2 @@
+website: https://geogeeks.org
```
Comment (samwilson) [inline _config.yml:1]: Should this be `baseurl`? https://jekyllrb.com/docs/configuration/options/#build-command-options

Comment (VeckoTheGecko) [reply _config.yml:1]: No. Looks there is some inconsistent terminology between the `jekyll build` command and the site config file. Using `website` works, but `baseurl` doesn't. This is also outlined in the article linked above.

Comment (VeckoTheGecko) [reply _config.yml:1]: A bit confusing 😅

Comment (samwilson) [reply _config.yml:1]: ah right! cool.

**Pull request merged**